lang/cs/Org.Apache.REEF.Client.Tests/packages.config

```diff
@@ -1,31 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
True, I deleted the files but forgot to remove references to them from .csproj files